packages/apps/graph/package.json

```diff
@@ -44,7 +43,6 @@
     "@types/node": "^16.0.0",
     "embedded-postgres": "~15.1.0-beta.6",
     "prettier": "~2.8.8",
-    "sort-package-json": "~2.4.1",
-    "ts-node": "~10.8.2"
+    "sort-package-json": "~2.4.1"
```

Comment:
We can replace `sort-package-json` with https://github.com/kellyselden/eslint-plugin-json-files/blob/master/docs/rules/sort-package-json.md